src/APNSClient.php

```diff
@@ -95,56 +72,15 @@ private function enqueueRequest( APNSRequest $request ): void {
 	 */
 	private function sendQueuedRequests(): array {
 
-		$responses = [];
-
-		do {
-			$status = curl_multi_exec( $this->curl_handle, $running_operation_count );
-
-			while ( true ) {
-
-				$info = curl_multi_info_read( $this->curl_handle );
-
-				if ( ! $info ) {
-					break;
-				}
-
-				if ( $info['result'] !== CURLE_OK ) {
-					throw new Exception( 'Request failed: ' . $info['result'] );
-				}
+		$network_responses = $this->network_service->sendQueuedRequests();
```

Comment:
```suggestion
		return $this->network_service->sendQueuedRequests();
```

I don't think we need to loop through the array below this line now that we're getting an array of `APNSResponse` back, right?

Reply:
Actually – looking at this again, I think the whole method could go and this line could be moved up into `sendRequests`

Reply:
I'll look into this.

Reply:
Done, thanks for catching that.

I was deep into the mechanical refactors or removing `Request` and `Response` and missed the bigger picture of what the method did.